src/shared/ui/GenreSelect/GenerMockData.js

```diff
@@ -0,0 +1,3 @@
+import react from "react"
+
+export const generList=['Action', 'Comedy', 'Drama', 'Fantasy'];
```

Comment:
let's add a space to get rid of this warning 
<img width="173" alt="image" src="https://github.com/GaKhalatyan86/ReactLessons/assets/36797396/897aba71-ba83-4bdd-99c8-6c7fe260fb1f">

Reply:
fixed